va/va.go

```diff
@@ -41,6 +42,10 @@ const (
 	// allowed accept up to 128 bytes before rejecting a response
 	// (32 byte b64 encoded token + . + 32 byte b64 encoded key fingerprint)
 	maxResponseSize = 128
+
+	// Constant for ALPN for TLS-ALPN-01 challenge
+	// https://rolandshoemaker.github.io/acme-tls-alpn/draft-ietf-acme-tls-alpn.html#rfc.section.3.1
```

Comment:
Could we switch out this URL for the canonical IETF url (https://tools.ietf.org/html/draft-ietf-acme-tls-alpn-01)